dart/packages/fury-core/lib/src/base_fury.dart

```diff
@@ -0,0 +1,15 @@
+import 'dart:typed_data';
+
+import 'meta/specs/custom_type_spec.dart';
+import 'package:fury_core/src/memory/byte_writer.dart';
+import 'package:fury_core/src/memory/byte_reader.dart';
+import 'ser/ser.dart' show Ser;
+
+abstract class BaseFury{
```

Comment:
```suggestion
abstract class BaseFury {
```
Do we need a space between class naem and `{`?